src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -163,6 +165,14 @@ def _get_microsoft_graph_resource_id(cloud_name):
     return graph_endpoint_mapper.get(cloud_name, None)
 
 
+def _get_storage_sync_endpoint(cloud_name):
+    storage_sync_endpoint_mapper = {
+        'AzureCloud': 'afs.azure.net',
+        'AzureUSGovernment': 'afs.azure.us',
```

Comment:
I think this is a duplication of https://github.com/Azure/azure-cli/blob/57596cdcf3463ae8c6fe5879f5f3ca95ba914711/src/azure-cli-core/azure/cli/core/cloud.py#L319

Maintaining 1 string in 2 places is not a good programming practice.

Reply:
It's a good suggestion. While I find two other functions in the file are also using the same way right now, so I'd prefer we stay the same code sytle until we decide to change all of them.